src/v2/components/PotentialFireList.jsx

```diff
@@ -154,13 +251,17 @@ export default function PotentialFireList(props) {
     return tidy
   }, [handlePotentialFire])
 
-  return <FireList
-    fires={fires}
-    firesByKey={firesByKeyRef.current}
-    indexOfOldFires={includesAllFires ? indexOfOldFires : -1}
-    nOldFires={indexOfOldFires > -1 ? allFiresRef.current.length - indexOfOldFires : 0}
-    onToggleAllFires={handleToggleAllFires}
-    region={region}
-    updateFires={updateFires}
-    {...props}/>
+  return 0,
+  <>
+    <Notification disableActiveWindow title="" {...notification}/>
```

Comment:
Previous PR, but seems like this is still missing the '={true}'

Reply:
`={true}` isn’t needed. A prop name without a value is boolean `true` by default (c.f., `disabled`).